tests/unit/store/utility.spec.ts

```diff
@@ -0,0 +1,282 @@
+      ];
+      const accentPhrases2 = [
+        createDummyAccentPhrase(["あ", "い", "う"]),
+        createDummyAccentPhrase(["え", "お"]),
+      ];
+      const result = isAccentPhrasesTextDifferent(
+        accentPhrases1,
+        accentPhrases2
+      );
+      expect(result).toBe(false);
+    });
+  });
+
+  test("buildAudioFileNameFromRawData", () => {
+    const fileNamePattern = "テストパターン";
+    const vars = {
+      index: 1,
+      characterName: "四国めたん",
+      text: "テストテキスト",
+      styleName: "ツンツン",
+      date: currentDateString(),
+    };
+    const result = buildAudioFileNameFromRawData(fileNamePattern, vars);
+    expect(result).toBe("テストパターン");
+  });
+
+  test("getToolbarButtonName", () => {
+    expect(getToolbarButtonName("PLAY_CONTINUOUSLY")).toBe("連続再生");
+    expect(getToolbarButtonName("STOP")).toBe("停止");
+    expect(getToolbarButtonName("EXPORT_AUDIO_SELECTED")).toBe(
+      "選択音声を書き出し"
+    );
+    expect(getToolbarButtonName("EXPORT_AUDIO_ALL")).toBe("全部書き出し");
+    expect(getToolbarButtonName("EXPORT_AUDIO_CONNECT_ALL")).toBe(
+      "音声を繋げて書き出し"
+    );
+    expect(getToolbarButtonName("存在しないタグ" as ToolbarButtonTagType)).toBe(
+      undefined
+    );
+  });
+
+  describe("createKanaRegex", () => {
+    it("includeSeparationがtrueの場合、読点とクエスチョンも含む", () => {
+      const regex = createKanaRegex(true);
+      expect(regex.test("あいうえお、")).toBe(true);
+      expect(regex.test("かきくけこ？")).toBe(true);
+    });
+
+    it("includeSeparationがfalseの場合、読点とクエスチョンを含まない", () => {
+      const regex = createKanaRegex(false);
+      expect(regex.test("あいうえお、")).toBe(false);
+      expect(regex.test("かきくけこ？")).toBe(false);
+    });
+  });
+
+  test("convertHiraToKana", () => {
+    expect(convertHiraToKana("あいうえお")).toBe("アイウエオ");
+    expect(convertHiraToKana("かきくけこ")).toBe("カキクケコ");
+    expect(convertHiraToKana("さしすせそ")).toBe("サシスセソ");
+  });
+
+  test("convertLongVowel", () => {
+    expect(convertLongVowel("アー")).toBe("アア");
+    expect(convertLongVowel("イー")).toBe("イイ");
+    expect(convertLongVowel("ウー")).toBe("ウウ");
+    expect(convertLongVowel("エー")).toBe("エエ");
+    expect(convertLongVowel("オー")).toBe("オオ");
+  });
+
+  test("getBaseName", () => {
+    expect(getBaseName("/path/to/file.txt")).toBe("file.txt");
+    expect(getBaseName("/path/to/file")).toBe("file");
+    expect(getBaseName("file.txt")).toBe("file.txt");
+    expect(getBaseName("file")).toBe("file");
+  });
+
+  test("isOnCommandOrCtrlKeyDown", () => {
+    expect(isOnCommandOrCtrlKeyDown({ metaKey: true, ctrlKey: false })).toBe(
+      isMac
+    );
+    expect(isOnCommandOrCtrlKeyDown({ metaKey: false, ctrlKey: true })).toBe(
+      !isMac
+    );
+    expect(isOnCommandOrCtrlKeyDown({ metaKey: true, ctrlKey: true })).toBe(
+      true
+    );
+    expect(isOnCommandOrCtrlKeyDown({ metaKey: false, ctrlKey: false })).toBe(
+      false
+    );
+  });
+
+  test("joinTextsInAccentPhrases", () => {
+    const accentPhrase: AccentPhrase = {
```

Comment:
joinのテストだからいろいろくっつけるテストがいるかも？
```suggestion
    // FIXME: いろいろくっつける
    const accentPhrase: AccentPhrase = {
```

Reply:
（joinTextsInAccentPhrasesという関数名だったけど、ひとつだけだった！！）

Reply:
（テストでしか使ってなかったので消させていただきました！）